scripts/kfctl.sh

```diff
@@ -137,6 +140,10 @@ if [ "${COMMAND}" == "init" ]; then
             shift
             PROJECT=$1
             ;;
+        --zone)
```

Comment:
Can we have only one way to explicitly set zone? e.g. if we set it as a command line argument then we should not set it using an environment variable.

Reply:
@jlewi Sure, but we are already doing same practice for `--platform`, `PLATFORM`, `--project`, `PROJECT`, `--email` and `EMAIL` so I thought that's kind of expected pattern here 🙂 

Let me know what you think. But if we are taking one out in the favor of other I would choose the `--zone` over environment variable.

Let me know your thoughts.

Reply:
I'm in favor of moving things to flags.